test/powershell/Modules/PSDesiredStateConfiguration/configuration.Tests.ps1

```diff
@@ -1,6 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 Describe "DSC MOF Compilation" -tags "CI" {
+
+<# PowerShellGet does not yet support DSCResources #>
```

Comment:
If there is an issue to track this. Please add a link.

Reply:
https://github.com/PowerShell/PowerShellGet/issues/211